__tests__/hooks/useFriction.spec.tsx

```diff
@@ -0,0 +1,176 @@
+    const Component = () => {
+      const [props] = useFriction({
+        from: { opacity: 0 },
+        to: { opacity: 1 },
+      });
+
+      return <div data-testid="node" {...props} />;
+    };
+
+    const { getByTestId } = render(<Component />);
+
+    await waitForAnimation();
+
+    const node = getByTestId('node');
+    const currentOpacity = parseFloat(node.style.opacity);
+
+    expect(currentOpacity).toBeGreaterThan(0);
+    expect(currentOpacity).toBeLessThan(1);
+  });
+
+  it('should not animate the specified property if pause is set to true', async () => {
+    const Component = () => {
+      const [props] = useFriction({
+        from: { opacity: 0 },
+        to: { opacity: 1 },
+        pause: true,
+      });
+
+      return <div data-testid="node" {...props} />;
+    };
+
+    const { getByTestId } = render(<Component />);
+
+    await waitForAnimation();
+
+    const node = getByTestId('node');
+    expect(node).not.toHaveAttribute('style');
+  });
+
+  it('should begin animation if pause switches to true on a subsequent render', async () => {
+    const Component = ({ pause }) => {
+      const [props] = useFriction({
+        from: { opacity: 0 },
+        to: { opacity: 1 },
+        pause,
+      });
+
+      return <div data-testid="node" {...props} />;
+    };
+
+    const { getByTestId, rerender } = render(<Component pause />);
+
+    await waitForAnimation();
+
+    const node = getByTestId('node');
+    expect(node).not.toHaveAttribute('style');
+
+    // Re-render Component with pause set to false.
+    rerender(<Component pause={false} />);
+
+    await waitForAnimation();
+
+    const currentOpacity = parseFloat(node.style.opacity);
+
+    expect(currentOpacity).toBeGreaterThan(0);
+    expect(currentOpacity).toBeLessThan(1);
+  });
+
+  it('should animate a delayed animation after the specified delay has elapsed', async () => {
+    const Component = () => {
+      const [props] = useFriction({
+        from: { opacity: 0 },
+        to: { opacity: 1 },
+        delay: 1000,
+      });
+
+      return <div data-testid="node" {...props} />;
+    };
+
+    const { getByTestId } = render(<Component />);
+
+    await waitForAnimation();
+
+    // The animating element should not have animated yet, since waitForAnimation
+    // will only wait for half a second by default.
+    const node = getByTestId('node');
+    expect(node).not.toHaveAttribute('style');
+
+    // Wait an additional full second. By this point, 1500 seconds will have elapsed,
+    // meaning that the 1000 ms delay will be exceeded and the animation should be running.
+    await waitForAnimation({ timeout: 1000, waitForTimeout: 1500 });
+
+    const currentOpacity = parseFloat(node.style.opacity);
+
+    expect(currentOpacity).toBeGreaterThan(0);
+    expect(currentOpacity).toBeLessThan(1);
+  });
+
+  it('should not apply translateZ on an element animating the transform property if disableHardwareAcceleration is true', () => {
```

Comment:
nit, I might move this test up next to the other `translate` test just for more clarity on why this omission is meaningful 🤷